<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Prime Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="MasterWidget" name="masterWidget"/>
      <widget class="SlaveWidget" name="slaveWidget"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionMaster"/>
    <addaction name="actionSlave"/>
   </widget>
   <addaction name="menuMode"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionMaster"/>
   <addaction name="actionSlave"/>
  </widget>
  <action name="actionMaster">
   <property name="text">
    <string>Master Mode</string>
   </property>
  </action>
  <action name="actionSlave">
   <property name="text">
    <string>Slave Mode</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MasterWidget</class>
   <extends>QWidget</extends>
   <header>masterwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SlaveWidget</class>
   <extends>QWidget</extends>
   <header>slavewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
